src/test/java/ToRemoveTest.java

```diff
@@ -0,0 +1,74 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ToRemoveTest {
+
+    @Test
+    @DisplayName("testToUpperCase")
+    void toUpperCase() {
+        final String TEST_STRING = "test";
+        String actualResult = ToRemove.toUpperCase(TEST_STRING);
+        String expectedResult = TEST_STRING.toUpperCase(Locale.ROOT);
```

Comment:
No. In unit test - please avoid such auxiliary calls. Simply `...expected = "TEST"`.